docs/README.md

```diff
@@ -1,49 +1,54 @@
 ---
-title: Introduction
+title: Quick start
 order: 0
 ---
 
-# Introduction
+# Quick start
 
-`urql` is an implementation of a GraphQL client, built to be both easy to use for newcomers to
-GraphQL, as well as extensible, to grow to support dynamic single-app applications and highly
-customized GraphQL infrastructure. In short, `urql` prioritizes usability and adaptability.
+Here we'll see the quickest way to get started with `urql`.
```

Comment:
I think we need to retain some sort of selling point / key factor as the first thing that users see:

>`urql` is a blazingly fast, lightweight, and customisable GraphQL client. We believe in usability and adaptability; `urql` is built with extensibility at its core, allowing you to to include just the key logic for a basic app, in addition to being able to grow to support dynamic single-app applications and highly customised infrastructure.
>
>In this page we'll take a look at the quickest route to get up and running with `urql`.

?